src/main/java/at/hannibal2/skyhanni/utils/ComputerTimeOffset.kt

```diff
@@ -82,8 +84,8 @@ object ComputerTimeOffset {
         val expectedDuration = 1.seconds
         val deviation = timeDifference - expectedDuration
 
-        if (deviation.absoluteValue > 100.milliseconds && config) {
-            ChatUtils.chat("Offset changed from ${lastDetectedOffset.format()} to ${deviation.format()}")
+        if (deviation.absoluteValue > 1.seconds && config) {
```

Comment:
deviation is normally around -1ms to 1ms. no need to increase the check range

Reply:
during joining hypixel I got 3 triggers that reached up to 0.7s

Reply:
> during joining hypixel I got 3 triggers that reached up to 0.7s

yeah this happens for me when i first click the multiplayer button, and then while im connecting and disconnecting from a server.